packages/jest-config/src/index.ts

```diff
@@ -313,9 +307,23 @@ export function readConfigs(
 
         return true;
       })
-      .map((root, projectIndex) =>
-        readConfig(argv, root, true, configPath, projectIndex),
-      );
+      .map((root, projectIndex) => {
+        // Skip on all except: projectIndex === 0, projects[0] === process.cwd() or realpath(process.cwd()) on Windows, and projects.length === 1
```

Comment:
Can you remove this comment? It doesn't add too much value as it duplicates the code below

Reply:
And add a changelog please! 😅

Reply:
Ok!